apps/docs/src/guide/cli/commands.md

```diff
@@ -76,8 +77,9 @@ npx fuels build --deploy
 
 Using the `--deploy` flag will additionally:
 
+1. Always compile contracts in `--release` mode
 1. Auto-start a short-lived `fuel-core` node if _needed_ ([docs](./config-file.md#autostartfuelcore))
-2. Run `deploy` on that node
+1. Run `deploy` on that node
```

Comment:
Numbering is incorrect.

Reply:
Great attention to detail! 🔍 

By consistently using `1.`, one can later reorder items freely without updating the numbering. It's a markdown feature, and the rendered docs will appear correctly:

![numbered-list](https://github.com/FuelLabs/fuels-ts/assets/26660/623edf2d-36c7-42fc-9383-44b8aed42e7e)


You can also see it's like in other places. This was an exception.

https://github.com/FuelLabs/fuels-ts/blob/c2624e51e44b2aa983877ede2815861312c84c5b/apps/docs/src/guide/cli/commands.md?plain=1#L71-L72

https://github.com/FuelLabs/fuels-ts/blob/c2624e51e44b2aa983877ede2815861312c84c5b/apps/docs/src/guide/cli/commands.md?plain=1#L92-L93

https://github.com/FuelLabs/fuels-ts/blob/c2624e51e44b2aa983877ede2815861312c84c5b/apps/docs/src/guide/cli/commands.md?plain=1#L119-L121